recipes/opencv/4.x/conanfile.py

```diff
@@ -158,6 +162,19 @@ def _patch_opencv(self):
         tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "ANDROID OR NOT UNIX", "FALSE")
         tools.replace_in_file(os.path.join(self._source_subfolder, "modules", "imgcodecs", "CMakeLists.txt"), "JASPER_", "Jasper_")
 
+        if self.options.dnn:
+            find_protobuf = os.path.join(self._source_subfolder, "cmake", "OpenCVFindProtobuf.cmake")
+            # variables generated by protobuf recipe have all lowercase prefixes
+            tools.replace_in_file(find_protobuf,
+                                  'find_package(Protobuf QUIET)',
+                                  '''find_package(Protobuf QUIET)
+            set(Protobuf_LIBRARIES ${protobuf_LIBRARIES})  # patch: map conan generated v ariable
+            set(Protobuf_LIBRARY ${protobuf_LIBS})  # patch: map conan generated v ariable
+            set(Protobuf_INCLUDE_DIR ${protobuf_INCLUDE_DIR})  # patch: map conan generated v ariable''')
```

Comment:
```suggestion
            set(Protobuf_LIBRARIES ${protobuf_LIBRARIES})  # patch: map conan generated v ariable
            set(Protobuf_LIBRARY ${protobuf_LIBS})  # patch: map conan generated v ariable
            set(Protobuf_INCLUDE_DIR ${protobuf_INCLUDE_DIR})  # patch: map conan generated v ariable''')
```

I would add a condition for each variable like `if(NOT DEFINED Protobuf_LIBRARIES)`, because protobuf recipe might be fixed later, and this patch would break in OpenCV.